### PR TITLE
HUB-396 upgrade opensaml to 3.4.0

### PR DESCRIPTION
**Pull request by JohnWattsGDS**
Update build.gradle to use build 162 of verify-saml-libs.

Adjust import in two tests to reflect new opensaml package structure.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,10 +3,11 @@ buildscript {
         if (System.getenv('VERIFY_USE_PUBLIC_BINARIES') == 'true') {
           logger.warn('Production builds MUST NOT be built with public binaries.\nUse artifactory/whitelisted-repos for production builds.\n\n')
           maven { url 'https://dl.bintray.com/alphagov/maven-test' }
+          maven { url 'https://build.shibboleth.net/nexus/content/repositories/releases' }
           jcenter()
         }
         else {
-          maven { url 'https://artifactory.ida.digital.cabinet-office.gov.uk/artifactory/whitelisted-repos' }
+            maven { url 'https://artifactory.ida.digital.cabinet-office.gov.uk/artifactory/whitelisted-repos' }
```
[the remark this answers is not in the document]
Comment (JohnWattsGDS) [reply build.gradle]: Fixed.

File: build.gradle [hunk 2]
```diff
@@ -3,6 +3,7 @@ buildscript {
         if (System.getenv('VERIFY_USE_PUBLIC_BINARIES') == 'true') {
           logger.warn('Production builds MUST NOT be built with public binaries.\nUse artifactory/whitelisted-repos for production builds.\n\n')
           maven { url 'https://dl.bintray.com/alphagov/maven-test' }
+          maven { url 'https://build.shibboleth.net/nexus/content/repositories/releases' }
```
Comment (chrisclayson) [inline build.gradle]: Part of me thinks that this shouldn't be necessary ... should our dependencies be published somewhere ... I'll have a look in the team manual.

Comment (JohnWattsGDS) [reply build.gradle]: Jakub said that Travis builds use the public repositories.

Comment (chrisclayson) [reply build.gradle]: Slight confusion on my part, this seems to be only used for the Travis builds, so its OK.

**Pull request merged**